vscode-dotnet-runtime-library/src/extension.ts

```diff
@@ -67,9 +74,10 @@ export function activate(context: vscode.ExtensionContext, parentExtensionId: st
         acquisitionInvoker: new AcquisitionInvoker(context.globalState, eventStream),
         versionResolver: new VersionResolver(context.globalState, eventStream),
         installationValidator: new InstallationValidator(eventStream),
+        timeoutValue: timeoutValue === undefined ? 120 : timeoutValue,
```

Comment:
if timeoutValue is undefined that typically means the user provided an improper value for it right? Asking because in the config you specify a default value. If the above is true about the user providing an improper value it'd probably be good to log that the value is unexpected so the user can fix it.

Reply:
In my testing the timeout value was undefined if the user doesn't specify one so I think the default value in the package.json is for display purposes when the user is editing their settings. Is that not the case?

Reply:
oh interesting, I definitely would not have thought so. But you trying it out and seeing differently is the most telling 😄